0_weatherbench2zarr.py

```diff
@@ -0,0 +1,55 @@
+"""
+Saving a one year subset (with select data variables) of
+[WeatherBench2](https://weatherbench2.readthedocs.io/en/latest/data-guide.html)
+to a local Zarr store.
+
+    python 0_weatherbench2zarr.py
+
+Full dataset(s) can be downloaded from
+https://console.cloud.google.com/storage/browser/weatherbench2
+"""
+import os
+
+import xarray as xr
+
+# %%
+# Temporal subset of WeatherBench2 dataset to 2 years
+store_name: str = "2020-full_37-6h-0p25deg-chunk-1_zuv500.zarr"
+if not os.path.exists(path=store_name):
+    # Open WeatherBench2 Zarr store from Google Cloud Storage
+    ds: xr.Dataset = xr.open_dataset(
+        filename_or_obj="gs://weatherbench2/datasets/era5/1959-2022-full_37-6h-0p25deg-chunk-1.zarr-v2",
+        engine="zarr",
+        chunks="auto",
+        consolidated=True,
+    )
+
+    # Subset to year 2000, 500hPa pressure level
+    ds_500hpa = ds.sel(time=slice("2000-01-01", "2000-12-31"), level=500, drop=True)
+
+    # Get data variable z500, u500, v500
+    ds_500hpa_zuv = ds_500hpa.get(
+        key=["geopotential", "u_component_of_wind", "v_component_of_wind"]
+    )
+
+    # Disable LZ4 compression, since cupy-xarray cannot handle it yet
+    for var in ds_500hpa_zuv.variables:
+        ds_500hpa_zuv[var].encoding["compressor"] = None
+
+    # Save to Zarr with chunks of size 1 along time dimension
+    # Can take about 1 hour to save 10.7GB of data at 40MB/s
```

Comment:
Oops, forgot to update this line :sweat_smile:

Benchmark download time at commit c63bc431858995ebdffdab1707480917ffc0e280 for 10.7GB of data was:

```
real	78m14.295s
user	25m23.062s
sys	5m0.360s
```

At commit a904131b45e62500451eafd0281638db565e875d, dataset is 18.2GB and took:

```
real	793m42.489s
user	279m16.092s
sys	51m52.820s
```
Internet speeds on my side was averaging at around 40MB/s :slightly_smiling_face: